pkgs/top-level/python-packages.nix

```diff
@@ -7312,6 +7312,10 @@ in {
 
   rxv     = callPackage ../development/python-modules/rxv     { };
 
+  executing = callPackage ../development/python-modules/executing { };
+
+  icecream = callPackage ../development/python-modules/icecream { };
```

Comment:
please attempt to sort them :)

Reply:
Ok, I have sorted them the best I could. Earlier when coding the first unsorted version, my first thought was put the packages ordered by name, but the list is not really sorted. Some places have a semblance of order, only to be followed by some unsorted names, and then some ordered names again... Then, as the instructions at the file beginning say nothing about sorting, I just put the names at the end. :-)

I think that to avoid this kind of confusion, if the intent is have a sorted list, that should be written on file top instructions and the out of place names be reordered. Should I open an issue to discuss about that further?